nbbuild/build.xml

```diff
@@ -19,7 +19,7 @@
     under the License.
 
 -->
-<project name="nbbuild" default="build-nozip" basedir="." xmlns:if="ant:if" xmlns:unless="ant:unless">
+<project name="nbbuild" default="build" basedir="." xmlns:if="ant:if" xmlns:unless="ant:unless">
```

Comment:
@matthiasblaesing what is the build output you're interested in? The contents of the `nbbuild/netbeans` folder is a source of some confusion. For a basic source build, people should be pointed at the zip. Anyone who doesn't need it can pass build-nozip?

Reply:
I'm not too bothered either way, as long as the build instructions are updated in the readme to reflect that an end user who wants to build from source needs to use `ant build` and should use the zip.  That's how we ended up with JavaFX in the Snap at one point!

I don't necessarily agree the default needs to change.  It depends how much we expect an end user to expect `ant` to do the right thing for them?  Funnily enough, when developing NetBeans I always end up running `ant build` out of habit anyway...

Reply:
@matthiasblaesing i don't care either way tbh. I always use `ant build-nozip` in my scripts and even CI uses that since I honestly always thought that the default would produce a zip, but as @neilcsmith-net mentioned at https://github.com/apache/netbeans-website/pull/643#pullrequestreview-1314473064 we have to either update the README or the default target.

I let you two decide, just tell me what to do :)

as I mentioned in https://github.com/apache/netbeans/pull/5568#issuecomment-1446222425, this was mostly a test run to check if changing the default would break something in CI.

Reply:
but reading between lines makes me think that you both want to revert that change and update the readme to `ant build` instead. Let me do that - has less of an impact which is usually the better option when nobody really cares :)